skimage_weekly_update.py

```diff
@@ -1,109 +1,214 @@
 #!/usr/bin/env python
+
+"""Print a weekly report for scikit-image repo activity in Markdown format."""
+
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#   "pygithub>=2.3",
+#   "tqdm",
+# ]
+# ///
+
+import sys
 import argparse
+import dataclasses
 from github import Github
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
+from tqdm import tqdm
+
+
+def item_state_as_emoji(item):
```

Comment:
@stefanv, I could remove this function if you think the emojis are more confusing that helpful. 

---

For others, the critique was that while they show the current open/closed state, that state isn't updated anymore once posted and may go stale quickly.